packages/js/customer-effort-score/src/customer-effort-score.tsx

```diff
@@ -27,14 +36,14 @@ const noop = () => {};
  * @param {Function} props.onModalShownCallback      Function to call when the modal is shown.
  * @param {Object}   props.icon                      Icon (React component) to be shown on the notice.
  */
-function CustomerEffortScore( {
+const CustomerEffortScore: React.FC< CustomerEffortScoreProps > = ( {
```

Comment:
We can use React.VFC here since this component doesn't render children

https://dev.to/aurora2500/don-t-use-function-components-use-void-function-components-493p

Reply:
Thanks. Updated in eb5d9b82fa7b87c445e002eb3e04492e9d2c1e1f.